mods/persistence/modules/world_save/saved_vars/saved_skipped.dm

```diff
@@ -0,0 +1,33 @@
+/////////////////////////////////////////
+// Things Not Saved
+/////////////////////////////////////////
+//Mark things as not saved that couldn't go anywhere else.
+//#TODO: Move things to respective files if they have one.
+
+/mob/observer
+	should_save = FALSE
+
+/atom/movable/openspace
+	should_save = FALSE
+/mob/living/simple_animal/hostile/mimic
+	should_save = FALSE
+
+/datum/extension/penetration
```

Comment:
The extension overrides are unnecessary, should_save is set to FALSE by default for extensions.

Reply:
I've had a problem with that. They would save and cause problems.

Reply:
That doesn't really make sense, should_save on the base type ``/datum/extension`` is FALSE, unless it's set elsewhere for some bizarre reason. At a glance though, it shouldn't be.